float64 x
float64 y
float64 z
float64 rot_x
float64 rot_y
float64 rot_z

---
bool success
